.github/workflows/test.yml

```diff
@@ -13,47 +13,57 @@ jobs:
     runs-on: ubuntu-22.04
     timeout-minutes: 20
     steps:
-    - uses: actions/setup-go@v3
-      with:
-        go-version: 1.19.x
-    - uses: actions/checkout@v3.1.0
-      with:
-        path: src/github.com/containerd/nerdctl
-        fetch-depth: 100
-    - uses: containerd/project-checks@v1.1.0
-      with:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
+      - uses: actions/checkout@v3.1.0
+        with:
+          path: src/github.com/containerd/nerdctl
+          fetch-depth: 100
+      - uses: containerd/project-checks@v1.1.0
+        with:
+          working-directory: src/github.com/containerd/nerdctl
+      - run: ./hack/verify-no-patent.sh
         working-directory: src/github.com/containerd/nerdctl
-    - run: ./hack/verify-no-patent.sh
-      working-directory: src/github.com/containerd/nerdctl
 
   golangci-lint:
     runs-on: ubuntu-22.04
     timeout-minutes: 20
     steps:
-    - uses: actions/checkout@v3.1.0
-      with:
-        fetch-depth: 1
-    - uses: actions/setup-go@v3
-      with:
-        go-version: 1.19.x
-    - name: golangci-lint
-      uses: golangci/golangci-lint-action@v3.3.1
-      with:
-        version: v1.50.1
-        args: --verbose
+      - uses: actions/checkout@v3.1.0
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3.3.1
+        with:
+          version: v1.50.1
+          args: --verbose
+
+  yaml-lint:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v3.1.0
+        with:
+          fetch-depth: 1
+      - name: Run yamllint
+        run: yamllint .
```

Comment:
golangci-lint and yaml-lint can be merged into a single "lint" job.

Also, could you squash the commits to 2 commits or 1 commit?

Reply:
I am squashing just trying to resolve all the YAML errors and correct the configuration.